docs/builder.rst

```diff
@@ -92,16 +104,57 @@ By default, the manifest builder does *not* perform any validation that the gene
    ethpm.exceptions.ValidationError: Manifest invalid for schema version 2. Reason: '_InvalidPkgName' does not match '^[a-z][-a-z0-9]{0,255}$'
 
 
+To write a manifest to disk
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+   to_disk(
+       manifest_root_dir: Optional[Path],
+       manifest_name: Optional[str],
+       prettify: Optional[bool],
+   )
+
+
+Writes the active manifest to disk. Will not overwrite an existing manifest with the same name and root directory.
+
+Defaults
+- Writes manifest to cwd unless a ``Path`` is provided as manifest_root_dir.
```

Comment:
change `cwd` to ``os.getcwd`` or expand to be full "current working directory (as returned by ``os.getcwd()``)"